src/commands/kv/namespace/create.rs

```diff
@@ -21,7 +27,22 @@ pub fn create(target: &Target, user: GlobalUser, title: &str) -> Result<(), fail
     });
 
     match response {
-        Ok(success) => message::success(&format!("Success: {:#?}", success.result)),
+        Ok(success) => {
+            message::success(&format!("Success: {:#?}", success.result));
+            match env {
+                Some(env) => message::success(&format!(
+                    "Add the following to your TOML under [env.{}]:",
+                    env
+                )),
+                None => message::success(&format!("Add the following to your TOML:")),
+            };
+            println!(
+                "kv-namespaces = [ \n\
+                 \t {{ binding: \"{}\", id: \"{}\" }} \n\
+                 ]",
```

Comment:
use `r##""##` strings here. see https://github.com/cloudflare/wrangler/blob/master/src/settings/target/mod.rs#L338

Reply:
So for the r##<text>## point, something fascinating i just learned is that
using that notation requires you to put the text into a separate variable
you can't use a separate variable for a formatting string in rust
All calls to println! and format! require string literals as the first argument!
So I'm not actually sure if we could use r##<text>##, as much as I wish it would work